src/components/SQForm/SQFormResetButtonWithConfirmation.tsx

```diff
@@ -1,8 +1,24 @@
 import React from 'react';
-import PropTypes from 'prop-types';
-import {useDialog} from '@selectquotelabs/sqhooks';
 import {DialogAlert, RoundedButton} from 'scplus-shared-components';
 import {useFormButton} from './useFormButton';
+import {useDialog} from '@selectquotelabs/sqhooks';
+
+type SQFormResetButtonWithConfirmationTypes = {
```

Comment:
I'm not sure it matters too much but I think all the other components so far have used `interface` instead of `type`. Also the names we're using are of hte format `${componentName}Props`